app/src/main/java/com/kamron/pogoiv/logic/PokemonNameCorrector.java

```diff
@@ -17,33 +17,18 @@
  */
 public class PokemonNameCorrector {
     private final PokeInfoCalculator pokeInfoCalculator;
-    private final HashMap<String, String> userCorrections;
 
     public PokemonNameCorrector(PokeInfoCalculator pokeInfoCalculator, Map<String, String> storedUserCorrections) {
```

Comment:
You should delete the last parameter. Also, you should be able to delete these lines from Pokefly onCreate()

         sharedPref = getSharedPreferences(PREF_USER_CORRECTIONS, Context.MODE_PRIVATE);
         corrector = initCorrectorFromPrefs(pokeInfoCalculator, sharedPref);

Github isn't allowing me to comment on those lines.

When deleting those line, I'm wondering if you should just force empty "PREF_USER_CORRECTIONS" and store it so that we wipe that data from "disk" for a few versions. Otherwise, that file is going to stay there till the user clears the app data or reinstalls it.

Also check for more code that can be deleted.